DataFormats/HLTReco/src/classes_def.xml

```diff
@@ -41,15 +41,20 @@
   <class name="trigger::TriggerObject" ClassVersion="10">
    <version ClassVersion="10" checksum="2923979599"/>
   </class>
+
   <class name="trigger::TriggerObjectCollection"/>
-  <class name="trigger::TriggerRefsCollections" ClassVersion="16">
+  <class name="trigger::TriggerRefsCollections" ClassVersion="18">
+   <version ClassVersion="18" checksum="2974272653"/>
+   <version ClassVersion="17" checksum="3691847911"/>
```

Comment:
Here and below, please don't include intermediate versions (e.g. `17`). The net result here should be that the latest class is `17` with the checksum `2974272653`, I think.

Reply:
We usually simply copy the classes_def.xml that is autogenerated. But since doing what you say works, we can do it.